MAINTAINERS.yml

```diff
@@ -5294,6 +5294,8 @@ West:
   status: maintained
   maintainers:
     - ssekar15
+    - cyliangtw
+    - ccli8
```

Comment:
Is @ssekar15 still active in this area? If not, I would suggest adding him as a collaborator instead since having 3 maintainers (with the permission to merge module PRs) for a module is a bit excessive.